test/test.js

```diff
@@ -408,6 +408,28 @@ describe('VendNumber', () => {
     })
   })
 
+  describe('#sumBy', () => {
+    it('should return 0.00 for non-numeric values', () => {
```

Comment:
You say `0.00` in the description here but test to 1.d.p. Perhaps leave out the `, 1` from each test so that we're testing the 2.d.p-by-default behaviour too?